src/github.com/getlantern/flashlight/config/bootstrap.go

```diff
@@ -121,7 +134,7 @@ func bootstrapPath(fileName string) (string, error) {
 	}
 	fullPath := filepath.Join(yamldir, fileName)
 	log.Debugf("Opening bootstrap file from: %v", fullPath)
-	return fullPath, nil
+	return yamldir, fullPath, nil
```

Comment:
maybe we don't need to return the fullPath anymore? it can be derived from the parameter and the return value.